opennms-correlation/drools-correlation-engine/src/test/opennms-home/etc/drools-engine.d/nodeParentRules/NodeParentRules.drl

```diff
@@ -65,18 +66,10 @@ rule "resolveIssue"
 		retract( $up );
 end
 
-function Cause possibleCause( DroolsCorrelationEngine engine, Long nodeId, Event symptom, Long POLL_INTERVAL ) {
+function Cause createPossibleCause( DroolsCorrelationEngine engine, Long nodeId, Event symptom, Long POLL_INTERVAL ) {

```

Comment:
Had to rename this function since the updated Drools compiler would confuse the function named `possibleCause` with the class named `PossibleCause`.

Reply:
(sigh)